src/sentry/event_manager.py

```diff
@@ -905,19 +908,60 @@ def get_culprit(data):
     )
 
 
-def _save_aggregate(event, flat_hashes, release, **kwargs):
+def _find_hashes(
```

Comment:
Should we rename `_find_hashes()` to `_find_existing_group_id()`?

Reply:
ok